grpc/ra-wrappers.go

```diff
@@ -121,15 +121,8 @@ func (rac RegistrationAuthorityClientWrapper) AdministrativelyRevokeCertificate(
 	return nil
 }
 
-func (ras *RegistrationAuthorityClientWrapper) NewOrder(ctx context.Context, request *rapb.NewOrderRequest) (*corepb.Order, error) {
-	resp, err := ras.inner.NewOrder(ctx, request)
-	if err != nil {
-		return nil, err
-	}
-	if resp == nil || !orderValid(resp) {
-		return nil, errIncompleteResponse
-	}
-	return resp, nil
+func (rac *RegistrationAuthorityClientWrapper) NewOrder(ctx context.Context, request *rapb.NewOrderRequest) (*corepb.Order, error) {
```

Comment:
This line renames `ras` to `rac`. That's good (I have no idea why it is `ras` elsewhere) but violates the standard of "all methods with the same receiver type should give the same name to that receiver". I'd leave it the same for now, and do a separate cleanup/rename PR.